docs/src/solvers/sde_solve.md

```diff
@@ -37,7 +37,7 @@ a matrix in the `SDEProblem` type. A special form of non-diagonal noise,
 commutative noise, occurs when the noise satisfies the following condition:
 
 ```math
-\sum_{i=1}^d g_{i,j_1}(t,u) \frac{\partial g_{k,j_2}}{\partial x_i} = \sum_{i=1}^d g_{i,j_2}(t,x) \frac{\partial g_{k,j_1}}{\partial x_i}
+\sum_{i=1}^d g_{i,j_1}(t,x) \frac{\partial g_{k,j_2}(t,x)}{\partial x_i} = \sum_{i=1}^d g_{i,j_2}(t,x) \frac{\partial g_{k,j_1}(t,x)}{\partial x_i}
```

Comment:
BTW, did you ask whether there's a nice way to write this for `g` a matrix?

Reply:
I decided to keep my email short (to maximize reply probability) so I did not include that. But yeah, I double checked a couple of times and am pretty sure that the form that you came up with is correct.

Reply:
Can we add the matrix way of writing the conditions while we're at it?